tasks/Add-music.md

```diff
@@ -13,6 +13,12 @@ Requirements:
 - It should be music relevant to the theme, try not to add any songs
 - Please make sure to use the existing API.
 - The music should be your own work. Please make sure you have the rights to any music that you add.
+- the music of any tile stops playing when the player leaves not moves
+  choose the background music randomly
+- add a loudness to the background music in the object you store it. Some music may be loud, some music is quite silent. If we add         music   from different people, we want to control that in the game as they make music of different volume.
+  document different open-source software in the task that we can use.
```

Comment:
this can be formulated differently. as editing the task is not part of the task.